.github/workflows/validation.yaml

```diff
@@ -71,7 +71,7 @@ jobs:
 
   upload-release-assets:
     name: Upload report to release 🔼
-    needs: r-pkg-validation
+    needs: build-install-check
```

Comment:
The only time you want to upload release assets is when the release exists, so this would be incorrect.

Reply:
The better logic would be to identify if a release (not just the tag) exists and then upload this asset.

Reply:
good point I will rollback it and change conditions

Reply:
Please don't. I have already rolled back.

Reply:
Rollback PR: https://github.com/insightsengineering/r.pkg.template/pull/76